src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/pathverification/PathVerificationService.java

```diff
@@ -740,4 +740,14 @@ private long getSwitchPortSpeed(IOFSwitch sw, OFPort inPort) {
     private long getAvailableBandwidth(long speed) {
         return (long) (speed * islBandwidthQuotient);
     }
+
+    /**
+     * Convert milliseconds into nanoseconds, taking in account illegal (-1) latency value.
+     */
+    public static long scaleLatencyMsToNs(long latencyMs) {
```

Comment:
Shouldn't the method be private? I don't a reason for  PathVerificationService to expose such methods.

Reply:
yes, it should. There was a plan to use it outside, but this is not required anymore.